charts/karmada-operator/templates/karmada-operator-clusterrole.yaml

```
@@ -4,8 +4,29 @@ metadata:
   name: {{ include "common.names.fullname" . }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
 rules:
-  - apiGroups: ['*']
-    resources: ['*']
-    verbs: ["*"]
-  - nonResourceURLs: ['*']
+  - apiGroups: ["coordination.k8s.io"]
+    resources: ["leases"]
+    verbs: ["get", "create", "update"]
+  - apiGroups: ["operator.karmada.io"]
+    resources: ["karmadas"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["operator.karmada.io"]
+    resources: ["karmadas/status"]
+    verbs: ["update"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["create"]
+  - apiGroups: [""]
+    resources: ["nodes", "pods"]
+    verbs: ["list"]
+  - apiGroups: [""]
+    resources: ["namespaces"]
+    verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["secrets", "services"]
+    verbs: ["get", "create", "update", "delete"]
+  - apiGroups: ["apps"]
+    resources: ["statefulsets", "deployments"]
+    verbs: ["get", "create", "update", "delete"]
+  - nonResourceURLs: ["/healthz"]
```

```suggestion
  - nonResourceURLs: ["/healthz"] # used to check whether the karmada apiserver is health
```

add some comments would be better

---

Added.